test/users/aroonsharma/README

```diff
@@ -9,8 +9,35 @@ Modulo Unrolling in Chapel" in PGAS 2014.
 These benchmarks originally compared different distributions, including
 a custom versions of Cyclic and Block Cyclic. These remain in this
 directory as a historical artifact but have been disabled from the
-tests.
+tests: CyclicZipOpt.chpl, MyBlockCyclic.chpl
+
 
 They are CyclicZipOpt.chpl and MyBlockCyclic.chpl. See the paper
```

Comment:
Your added comment is repetitive with this paragraph.  Perhaps they should be combined?